### DISCUSSION TITLE
If my model is deleted, do I get a notification somewhere?

### DISCUSSION THREAD

**Discussion by PeterLorre** [civitai/civitai#390] [Q&A]
I no longer see one of my created LORA models since the recent changes on the site. I received no communication or notification about its removal or deletion. If it was removed by Civitai, why wouldn't I at least get an email with a reason? I see no incentive to keep making models if they just disappear into thin air. Thanks.

**Comment by Nenotriple** [comment]
What's likely happened is the content filter is preventing you from seeing your own models.

**Reply by PeterLorre** [reply]
That's what I assumed as well but all of my models are there except for one. I guess I'll have to just give it some time while they smooth out all of the new changes.

**Reply by Nenotriple** [reply]
Take a look in your account settings and double check that mature content is enabled. It looks like the setting is resetting, or sometimes not even saving.

**Comment by mrfood0** [comment]
This appeared to happen to me as well, until I set the filter on the top left from "Highest Rated" to "Newest" and all my models showed up again. So try that if you haven't.

**Comment by PeterLorre** [comment]
Thank you @mrfood0, I tried all of that before posting here. I just logged in and the model is back as well as many of the preview images that also disappeared from other models. Not sure what happed, I know their server went down yesterday but all is well again.
